app/views/faqs/index.html.erb

```diff
@@ -6,7 +6,7 @@
   <div class="usa-width-three-fourths usa-layout-docs-main_content">
     <h1><%= @page_title %></h1>
 
-    <p>Moves are complex elephants, so let’s break this process down into something digestible. When in doubt, give <%= link_to 'your local personal property office', locator_maps_path %> a call.  They are a wealth of knowledge and have the most up-to-date information to help you!</p>
+    <p>Moves are complex elephants, so let’s break this process down into something digestible. When in doubt, give <%= link_to 'your local personal property office', offices_path %> a call.  They are a wealth of knowledge and have the most up-to-date information to help you!</p>
```

Comment:
There are links to the locator maps page within faqs.yml as well - are they still correct?

Reply:
@jamesatheyDDS The URL for this page didn't change, so yes.